File: www/kd/cgi-codes/stat.py
from __future__ import print_function

import sys
import os

import cgi
import cgitb

form = cgi.FieldStorage()

code = form.getvalue("code", "200")

print("Content-Type: text/plain", end="\r\n");
print("Status: ", code, end="\r\n\r\n");
print("PYTHON Status: ", code);

File: www/kd/cgi-codes/stat.pl

use strict;
use warnings;
use CGI;

my $cgi = CGI->new;

my $code = $cgi->param('code');

print("Content-Type: text/plain\r\n");
print("Status: ", $code, "\r\n\r\n");

print("PERL Status: ", $code, "\r\n");

